mathics/builtin/control.py

```diff
@@ -759,9 +760,8 @@ def apply1(self, expr, evaluation):
         except WLThrowInterrupt as e:
             return e.value
         return ret
- 
+
     def apply3(self, expr, form, f, evaluation):
-        'Catch[expr_, form_, f__:Identity]'
```

Comment:
Why did you remove this line? This was for stsblish the default handler function to the identity. Shouldn't it be there?

Reply:
Ooops - definitely a mistake.  It should be back in d0be8daf   Thanks for catching.

Reply:
OK. Then, let's merge this. Over the weekend I can try to figure out what happens with the throw inside called functions